### PR TITLE
Reset styles before DM CSS

### PR DESCRIPTION
**Pull request by samuelhwilliams**
Move our DM CSS reset to the top of our `application.css` file so that we clear styling on all HTML core elements, giving us a definitively clean base upon which to buildn.

### CODE REVIEW

File: app/assets/scss/application.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Reset styles on core HTML elements so that we have a clean base for DM styles to build on.
```
Comment (idavidmcdonald) [inline app/assets/scss/application.scss]: Nice, might be nice to add something about browser added styles as those are the reason we have a reset stylesheet. Apart from that I like this

Comment (samuelhwilliams) [reply app/assets/scss/application.scss]: Comment clarified. 👍

File: app/assets/scss/application.scss [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// Reset styles on core HTML elements that might be injected by eg a user's browser, so that we have a known clean base
```
Comment (idavidmcdonald) [inline app/assets/scss/application.scss]: Sorry, should it be `that might be injected eg by a user's browser`?

Comment (samuelhwilliams) [reply app/assets/scss/application.scss]: Either/or is fine.

**Pull request merged**